tests/test_utils_iterators.py

```diff
@@ -45,6 +46,54 @@ def test_xmliter_unusual_node(self):
                  for e in self.xmliter(response, 'matchme...')]
         self.assertEqual(nodenames, [['matchme...']])
 
+    def test_xmliter_unicode(self):
+        # example taken from https://github.com/scrapy/scrapy/issues/1665
+        body = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
hm, this test runs differently in Python 2 and 3 -- the input is bytes in Python 2 (with UTF-8 encoded data) and unicode in Python 3.
perhaps it should use the `u` prefix?